sunpy/lightcurve/sources/het.py

```diff
@@ -0,0 +1,96 @@
+    else:
+        type_of_data = 'other'
+
+    #Header
+    header = [  'Column 2: Electron flux, 0.7-1.4 MeV, particles/(cm2-sr-sec-MeV)', 
+                'Column 3: Uncertainty (sigma) for 0.7-1.4 MeV electron flux',
+                'Column 4: Electron flux, 1.4-2.8 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 5: Uncertainty (sigma) for 1.4-2.8 MeV electron flux',
+                'Column 6: Electron flux, 2.8-4.0 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 7: Uncertainty (sigma) for 2.8-4.0 MeV electron flux',
+                'Column 8: Proton flux, 13.6-15.1 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 9 Uncertainty (sigma) for 13.6-15.1 MeV proton flux',
+                'Column 10: Proton flux, 14.9-17.1 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 11: Uncertainty (sigma) for 14.9-17.1 MeV proton flux',
+                'Column 12: Proton flux, 17.0-19.3 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 13: Uncertainty (sigma) for 17.0-19.3 MeV proton flux',
+                'Column 14: Proton flux, 20.8-23.8 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 15: Uncertainty (sigma) for 20.8-23.8 MeV proton flux',
+                'Column 16: Proton flux, 23.8-26.4 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 17: Uncertainty (sigma) for 23.8-26.4 MeV proton flux',
+                'Column 18: Proton flux, 26.3-29.7 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 19: Uncertainty (sigma) for 26.3-29.7 MeV proton flux',
+                'Column 20: Proton flux, 29.5-33.4 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 21: Uncertainty (sigma) for 29.5-33.4 MeV proton flux' ,
+                'Column 22: Proton flux, 33.4-35.8 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 23: Uncertainty (sigma) for 33.4-35.8 MeV proton flux' ,
+                'Column 24: Proton flux, 35.5-40.5 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 25: Uncertainty (sigma) for 35.5-40.5 MeV proton flux' ,
+                'Column 26: Proton flux, 40.0-60.0 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 27: Uncertainty (sigma) for 40.0-60.0 MeV proton flux' ,
+                'Column 28: Proton flux, 60.0-100.0 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 29: Uncertainty (sigma) for 60.0-100.0 MeV proton flux' ]
+   
+
+    data_modify = []
+    month_dict = {'Jan':1,'Feb':2,'Mar':3,'Apr':4,'May':5,'Jun':6,'Jul':7,'Aug':8,'Sep':9,'Oct':10,'Nov':11,'Dec':12}
+
+
+    #Adding Time Column based on type of data
+    if type_of_data == '1min':
+        header = ['Verse Number', 'DateTime'] + header
+        
+        #Converting start datetime separate columns into a single datetime column
+        for i in range(len(data)): 
+            date = datetime(data['col2'][i], month_dict[ data['col3'][i] ], data['col4'][i], int(("%04d" % (data['col5'][i],))[:2]), int(("%04d" % (data['col5'][i],))[2:]) )
```

Comment:
we are not using this way of formatting for string (`%`)  anymore... but if you are then converting it to integer... what's the point?